cmake/UserOverride.cmake

```diff
@@ -17,21 +17,21 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
             set(common "${common} -DMS_WIN64")
         endif()
     endif()
-    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 -ffast-math -funroll-loops")
-    set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g")
+    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 -march=native -ffast-math -funroll-loops")
+    set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g -ggdb")
 elseif (CMAKE_CXX_COMPILER_ID STREQUAL "Intel")
     # icpc
-    set(common "-std=c++11 -Wall -fno-common")
+    set(common "-Wall -fno-common")
     set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -xHOST -O3 -no-prec-div")
     set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g -O0")
 elseif (CMAKE_CXX_COMPILER_ID MATCHES Clang)
     # clang
-    set(common "-std=c++11 -Wall -Wextra -fPIC")
-    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 -ffast-math -funroll-loops")
-    set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g")
+    set(common "-Wall -Wextra -Wno-unused-parameter -fPIC")
+    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 -march=native -ffast-math -funroll-loops")
+    set(CMAKE_CXX_FLAGS_DEBUG_INIT   "${common} -g -ggdb")
```

Comment:
This tunes code for gdb, and we'd want to use lldb together with Clang

Reply:
True, I am just doing what `master` is doing, which had this in `CMakeLists.txt`:
```cmake
if (CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
    set(CMAKE_CXX_FLAGS_DEBUG
        "${CMAKE_CXX_FLAGS_DEBUG} -Wno-unused-parameter -ggdb")
```
I would suggest we figure this out / fix this in a separate PR.

Reply:
Ah, okay. We'll leave it for another PR